src/vendors/types.ts

```diff
@@ -10,6 +10,7 @@ export type Vendor = {
   currencies: Array<string>
   address?: string
   phoneNumber?: string
+  cico?: boolean
```

Comment:
Could you split this into 'acceptsGuilder: boolean, providesGuilder: boolean'?

Reply:
Done!